mutator/expression/comparison.go

```diff
@@ -0,0 +1,47 @@
+package expression
+
+import (
+	"go/ast"
+	"go/token"
+	"go/types"
+
+	"github.com/zimmski/go-mutesting/mutator"
+)
+
+var tokenMutations = map[token.Token]token.Token{
+	token.LSS: token.LEQ,
+	token.LEQ: token.LSS,
+	token.GTR: token.GEQ,
+	token.GEQ: token.GTR,
+}
+
+func init() {
+	mutator.Register("expression/comparison", MutatorComparison)
+}
+
+// MutatorComparison implements a mutator to change comparisons.
+func MutatorComparison(pkg *types.Package, info *types.Info, node ast.Node) []mutator.Mutation {
+	n, ok := node.(*ast.BinaryExpr)
+
+	if !ok {
+		return nil
+	}
+
+	o := n.Op
+	r, ok := tokenMutations[n.Op]
+
+	if !ok {
+		return nil
+	}
+
+	return []mutator.Mutation{
+		{
```

Comment:
Since we define the type for every item in the other files, please use the type here too (`mutator.Mutation{`.

Reply:
(If you like to remove them, please do another PR with that change and we need an automated way to check misusage. However, I do not know of any linter that marks this as a problem. Do you?)

Reply:
It looks like `gofmt -s` will remove them, the only linter I'm aware of which will bring this up as a warning is [`golangci-lint`](https://github.com/golangci/golangci-lint).

Reply:
Then we can just use `gofmt -s` -> https://github.com/zimmski/go-mutesting/pull/67

However, the simplify option is not the default and I found lots of cases where the Go authors have conflicts.

Reply:
This hits a bug in gocode that comes with the Go plugin of VS Code. However, K8s uses this option [too](https://github.com/kubernetes/kubernetes/blob/6f897af2daffd8cd3539f1ef279b9f9bc280d98f/hack/verify-gofmt.sh#L56) so I guess the reasons I found that speak against including the option are mute.